bundle/src/main/java/com/adobe/acs/commons/redirects/models/RedirectRule.java

```diff
@@ -129,13 +170,13 @@ public Pattern getRegex() {
     }
 
     public ZonedDateTime getUntilDate() {
-        return untilDate;
+        return zonedUntilDate;
     }
 
     @Override
     public String toString() {
         return String.format("RedirectRule{source='%s', target='%s', statusCode=%s, untilDate=%s, note=%s, contextPrefixIgnored=%s}",

```

Comment:
Because all properties were included here before, you should do the same with the ```createdBy``` property and include it in the ``´toString()```  implementation.

Reply:
I removed the zonedUntilDate property to avoid confusion, the code can do without it.